modules/heat_conduction/src/actions/ThermalContactMaterialsAction.C

```diff
@@ -57,7 +57,8 @@ ThermalContactMaterialsAction::act()
 
   InputParameters params = _factory.getValidParams(type);
   // Extract global params
-  _app.parser().extractParams(_name, params);
+  if (isParamValid("parser_syntax"))
+    _app.parser().extractParams(getParam<std::string>("parser_syntax"), params);
```

Comment:
I know this isn't your fault but I've been wanting to get rid of this `extractParam` call in user code. The BISON guys started adding it years ago even though it's no longer really necessary. Just some history, this needs to go away eventually, but not necessarily in this PR.